src/plugins/mer/merbuildconfiguration.cpp

```diff
@@ -76,6 +96,26 @@ void MerBuildConfiguration::setUseExtraParserArguments(bool value)
     setupExtraParserArguments();
 }
 
+void MerBuildConfiguration::updateExtraParserArguments() const
+{
+    const MerSdk *const merSdk = MerSdkKitInformation::sdk(target()->kit());
+    QTC_ASSERT(merSdk, return);
+
+    if (!merSdk->connection()
```

Comment:
I would rather use QTC_ASSERT to check merSdk->connection(). Also, is it ever possible to get nullptr here?

Reply:
It would not be possible if `merSdk->connection()` return reference (or shared pointer) instead of plain pointer.:) Switched to QTC_ASSERT.